syft/frameworks/torch/tensors/interpreters/precision.py

```diff
@@ -489,6 +489,39 @@ def addmm(bias, input_tensor, weight):
 
         module.addmm = addmm
 
+        def sigmoid(tensor):
+            """
+            Overloads torch.sigmoid to be able to use MPC
+            Approximation with polynomial interpolation of degree 10 over [-10,10]
+            Ref: https://mortendahl.github.io/2017/04/17/private-deep-learning-with-mpc/#approximating-sigmoid
+            """
+
+            weights = [0.5, 0.216578258, -0.0083312848, 0.0001876528, -1.9669e-06, 7.5e-09]
+            degrees = [0, 1, 3, 5, 7, 9]
+
+            max_degree = degrees[2]
```

Comment:
shouldn't it be -1 instead of 2?

Reply:
It's temporally.
After merges from last week, these activation functions got too slow (probably due to several multiplications with degrees higher than 3), so I needed to reduce the max_degree to keep it still doable. The precision we lose by doing this is not that significant and I plan to get back to max_degree=9 after I investigate further the problem with this increase in time execution.